whatis277/harvest
# bean: Add basic subscriptions page

Adds basic subscriptions page

Primitively lists all subscriptions
User ID is hard coded

Testing instructions:
1. `dc down --volumes`
2. `dc up bean`
3. [`/get-started`](https://localhost/get-started)
4. Enter any valid email address
5. Ensure you're redirected to [`/subscriptions`](https://localhost/subscriptions)
6. Ensure you can the seeded data 

<img width="512" alt="image" src="https://github.com/whatis277/harvest/assets/5631091/8391ee4d-cdf8-4f74-b10f-ccdf56bfb9dd">
